renovate.json

```
@@ -17,6 +17,15 @@
       "matchUpdateTypes": ["minor"],
       "enabled": false
     },
+    {
+      "description": [
+        "Ignore tinymce upgrades to version 6",
+        "Tinymce version 6 doesn't support pasting in from Word documents so we must lock to version 5 for now"
+      ],
+      "matchPackagePatterns": ["tinymce"],
+      "versioning": ">5.10.7",
```

I think we can be a little more precise here by telling it to only ignore major updates (so we still get patches and minors):
```suggestion
      "matchUpdateTypes": ["major"],
```